packages/create-wmr/src/index.js

```diff
@@ -52,13 +52,13 @@ sade('create-wmr [dir]', true)
 			color: 'yellow',
 			text: 'installing WMR...'
 		}).start();
+
+		spinner.start('scaffolding new project...');
+		await scaffold(ctx);
+		spinner.succeed('project created!');
+
 		const packageManager = /yarn/.test(process.env.npm_execpath) ? 'yarn' : 'npm';
-		// @ts-ignore-next
-		await getPackageManager({ prefer: packageManager }).catch(() => {
-			process.stderr.write(`\n${red(`${packageManager} cannot be found`)}\n`);
-			process.exit(1);
-		});
```

Comment:
I believe I wrote this because installing with yarn was originally a flag, like used in `preact-cli`. This is irrelevant though with the current set up. You can't run `npm init` / `yarn create` without having npm/yarn available. NPX comes with NPM, so it too isn't an potential issue.